network.go

```diff
@@ -135,3 +157,34 @@ func getIPFromHeader(r *http.Request, name string) net.IP {
 
 	return nil
 }
+
+func extractSubFromJWT(jwtString string, secret string) (string, error) {
+	claims := &jwt.StandardClaims{}
+	token, err := jwt.ParseWithClaims(jwtString, claims, func(token *jwt.Token) (interface{}, error) {
+		return []byte(secret), nil
+	})
+	if err != nil {
+		return "", err
+	}
+	if !token.Valid {
+		return "", ErrInvalidJWT
+	}
+	return fmt.Sprint([]byte(claims.Subject)), nil
+}
+
+func getAuthorizationToken(r *http.Request) (string, bool) {
+	const bearer = "bearer "
```

Comment:
You are in the scope of your private function here. No need to declare `const` there since you don't expose this variable.

```suggestion
	bearer := "bearer "
```